randovania/server/discord/preset_lookup.py

```diff
@@ -173,15 +173,14 @@ async def reply_for_preset(message: discord.Message, versioned_preset: Versioned
 
 async def reply_for_layout_description(message: discord.Message, description: LayoutDescription):
     embed = discord.Embed(
-        title=f"Spoiler file for Randovania {description.randovania_version_text}",
+        title=f"Spoiler file (Generated in {description.randovania_version_text})",
     )
 
     if description.world_count == 1:
         preset = description.get_preset(0)
         embed.description = (
             f"{preset.game.long_name}, with preset {preset.name}.\n"
-            f"Seed Hash: {description.shareable_word_hash}\n"
-            f"Permalink: {description.permalink.as_base64_str}"
+            f"Seed Hash for {preset.game.long_name}: {description.shareable_word_hash}"
```

Comment:
https://github.com/randovania/randovania/issues/6645#issuecomment-2136793027

> seed hash for \<current version\>: \<current version seed hash\>

Reply:
fixed it, thought I had merged before the pr

Reply:
no, this isnt fixed.

Reply:
^, this is using the game name again instead of the current version